tests/serializers/base_serializer_test.py

```diff
@@ -31,6 +28,20 @@ class MockSerializer(BaseSerializer):
 
         expect(MockSerializer.call(mock_model)).to(have_keys(id=id_, username=username))
 
+    def test_call_on_array_of_models(self, faker):
```

Comment:
Ideally add a comment to the test classes/functions so everyone knows the exact purpose of the test (even if it's just reiterating the name). Sometimes the name implies something but the test is a bit more nuanced. Going to debug why your test is broken is really hard if I'm not 100% sure what the test is trying to prove. 
And the class has a function with a class within it!?! I haven't seen that, that's some inception shit! The tests are a bit hard for me to follow because of the flow of them, but I think it would be super clear with a short, 1 sentence description inside each class init and test function. Then I can basically just read the descriptions and not the code and get what all the tests do

Reply:
I'm following this pattern; https://github.com/testdouble/contributing-tests/wiki/Arrange-Act-Assert and this https://blog.testdouble.com/posts/2020-02-25-necessary-and-sufficient/

I'm trying to build a fully isolated test here so I built a custom class to test against.
The core of the test is ...

Does the serializer, when given an array, operate the same as the serializer when looped over each element of the array.